main.py

```diff
@@ -137,12 +138,15 @@ def one_ai():
     print("Starting the simulator with one AI")
     simulator.personPlusAi(cast.KillerOfWorlds)
     #dirname = os.path.dirname(__file__)
-    #script_input(os.path.join(dirname,"scripts/input/"))
+    #script_input(oxs.path.join(dirname,"scripts/input/"))
 
 def two_ai():
     watchTV = setting.InfiniteTelevision()
 
-    simulator.twoAiPlusPerson(cast.Avatar, cast.Unvatar)
+    with open(input("file path to load:")) as f:

```

Comment:
it'd be cool if this supported a CLI argument in addition to an interactive mode! 

- [ ] abstract this path into an argument to `two_ai`

Reply:
How do we do such a thing?